ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/ExceptionRequestModal.tsx

```diff
@@ -28,35 +30,54 @@ export type ExceptionRequestModalOptions = {
 
 export type ExceptionRequestModalProps = {
     type: CveExceptionRequestType;
+    isUpdate?: boolean;
+    id?: string;
     cves: ExceptionRequestFormProps['cves'];
     scopeContext: ScopeContext;
-    onExceptionRequestSuccess: (vulnerabilityException: BaseVulnerabilityException) => void;
+    onExceptionRequestSuccess: (vulnerabilityException: VulnerabilityException) => void;
     onClose: () => void;
 };
 
 function ExceptionRequestModal({
     type,
+    isUpdate = false,
+    id = '',
```

Comment:
`isUpdate` and `id` are two extra bits of information we pass to the `ExceptionRequestModal` to make the modal more reusable for the update flow